src/main/java/com/thebois/views/GameScreen.java

```diff
@@ -15,22 +15,26 @@
     private final ShapeRenderer batch;
     private final OrthographicCamera camera;
     private final FitViewport viewport;
-    private final float worldWidth = 800f;
-    private final float worldHeight = 480f;
-    private final float rectangleSize = 64f;
+    private final float worldWidth = 1000f;
+    private final float worldHeight = 1000f;
```

Comment:
Will `worldWidth` and `worldHeight` always be the same? If so, you could just have one `worldSize`.

Reply:
I don't see a downside to being able to have them as differnt sizes in the future. Pretty much all that is needed is another constructor in World.

So the question is should we leave it until then or implement it now or remove it for the time being?

Reply:
Ah wait, IIRC `worldWidth` and `worldHeight` control the size of the viewport in pixels, right?

And in `Main.java` we have
```java
    private static final int WIDTH = 800;
    private static final int HEIGHT = 480;
```
which are used to set the size of the application window. Should these four variables not be the same? If so, you should probably pass the `WIDTH` and `HEIGHT` from `Main` down through `ColonyManager` into `GameScreen`.